microsoft-edge/devtools-guide-chromium/javascript/overrides.md

```diff
@@ -17,81 +17,113 @@ You can now take a resource of the current webpage and store it locally.  When y
 
-In the Sources editor, adding the name of the file to the overrides list:
+   In the editor pane of the **Sources** tool, a page icon with a purple dot is added to the file's tab. The purple dot indicates that the file is a local file that overrides the file that's returned from the web server:
 
-![In the Sources editor, adding the name of the file to the overrides list](./overrides-images/javascript-overrides-file-name.png)
+   ![Storing the file in your overrides folder adds a purple dot to the page icon](./overrides-images/javascript-overrides-file-stored.png)
 
-Right-click a file name and then select **Save for overrides**:
 
-![Right-click a file name and then select 'Save for overrides'](./overrides-images/javascript-overrides-save-for-overrides.png)
+<!-- ====================================================================== -->
+## Changing the style by using the override file
+
+Continuing from above, you can now change the styles of the webpage by using your local override CSS file (in this example, `to-do-styles.css`).  Add a thick red border around the rendered webpage body, as follows:
+
+1. In DevTools, select the **Elements** (![Elements tool icon](./overrides-images/elements-tool-icon.png)) tool, and then make sure the **Styles** tab is selected.
 
-The file is stored in your overrides folder.  Verify that DevTools create a folder that is named using the URL of the file with the correct directory structure.  The file is stored inside.  The file name in the editor now also shows a purple dot that indicates that the file is local and not a live one.
+1. Copy the following CSS style property, and then paste it into the existing `body` element CSS rule that's in your CSS override file, such as `to-do-styles.css`:
 
-Successfully stored the file in your overrides folder:
+   ```css
+   border: 10px solid firebrick
+   ```
 
-![Successfully stored the file in your overrides folder](./overrides-images/javascript-overrides-file-stored.png)
+   ![Changing the webpage styles persistently by editing a file in your overrides folder](./overrides-images/javascript-overrides-changing-styles.png)
 
-In the following example, you can now change the styles of the webpage.  To add a red border around the file, on the **Styles** editor, copy the following style, and add it to the body element.
+   A thick red ("firebrick") border is added around the body of the rendered webpage, and the modified CSS file is automatically saved on your computer, in your Overrides directory.
 
-```css
-border: 10px solid firebrick
-```
+1. Refresh the webpage.  The thick red border remains displayed, and none of your work is lost, as it would have been if rendering the CSS file that's returned by the web server instead of using the local override file.
```

Comment:
```suggestion
1. Refresh the webpage.

   The thick red border remains displayed, and none of your work is lost, as it would have been if rendering the CSS file that's returned by the web server instead of using the local override file.
```